ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp

```diff
@@ -135,44 +130,43 @@ std::vector<Tensor> _rdiv_bw(
     const Tensor& grad, const Tensor& input, float scalar, string round_mode, const std::optional<MemoryConfig>& output_mem_config) {
     std::vector<Tensor> grad_tensor;
     TT_FATAL((round_mode == "None" || round_mode == "trunc" || round_mode == "floor") && "Incorrect rounding mode (expected 'None', 'trunc', or 'floor')");
-    auto output_memory_config = output_mem_config.value_or(input.memory_config()); //TODO: Remove after ternary forward ops migration is completed
     float t_nan = std::nanf("");
     float t_inf = std::numeric_limits<float>::infinity();
     if (round_mode == "None") {
-        Tensor result = where(
+        Tensor result = ttnn::where(
             ttnn::nez(input),
-            ttnn::multiply(ttnn::neg(grad, output_memory_config),
-                (ttnn::multiply(ttnn::reciprocal(ttnn::square(input, output_memory_config)), scalar, std::nullopt, output_memory_config)),
+            ttnn::multiply(ttnn::neg(grad, output_mem_config),
+                (ttnn::multiply(ttnn::reciprocal(ttnn::square(input, output_mem_config)), scalar, std::nullopt, output_mem_config)),
```

Comment:
Ideally this would just be memory_config as the name to stay consistent.

Reply:
I'll update the code on a separate PR once Unary bw ops are done